squid-mixin/config.libsonnet

```
@@ -1,16 +1,41 @@
 {
-  _config+:: {
-    dashboardTags: ['squid'],
-    dashboardPeriod: 'now-1h',
-    dashboardTimezone: 'default',
-    dashboardRefresh: '1m',
+  local this = self,
 
-    // alerts thresholds
-    alertsCriticalHighPercentageRequestErrors: 5,
-    alertsWarningLowCacheHitRatio: 85,
-    enableLokiLogs: true,
-    enableMultiCluster: false,
-    multiclusterSelector: 'job=~"$job"',
-    squidSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
+  // Basic filtering
+  filteringSelector: 'job=~"$job", instance=~"$instance"',
```

I think this is the incorrect filtering selector. I believe its default should probably be 
```suggestion
  filteringSelector: 'job="integrations/squid"',
```